### ISSUE TITLE
[Tizen.CircularUI] [CircularListView] Scroller size

### ISSUE THREAD

**Issue by jsobczuk** [Samsung/Tizen.CircularUI#123]
Hi

I have encountered strange behaviour of scroller while using CircularListView in my sample application. Issue is visible after going into page with CircularListView and not making any action. After moving to element below scroller width is set, but it's done only after some movement. I made an video and screenshot that shows this issue.

In my opinion size of the scroller should be set while initializing the page , not after action. It's done automatically and I cannot set scroller size after initializing page or from property in XAML file.

![scroller](https://user-images.githubusercontent.com/39958609/45752972-85ceea80-bc17-11e8-802e-ab18e90b65e2.png)

Also I attached video in .zip file. If it's necessary I will make a new task at JIRA.

[ListViewScrollerWrongSizing.zip](https://github.com/Samsung/Tizen.CircularUI/files/2397017/ListViewScrollerWrongSizing.zip)

Best regards
j.sobczuk

**Comment by jkpu** [comment]
This issue occured only in Tizen 4.0 emulator. Our Test target and Galaxy Watch has no this issue. and 5.0 emulator has no this issue. so I think this issue related emulator issue.
 Tizen.CircularUI reference device is real Galaxy watch device not emulator.  so close this issue.

**Comment by jkpu** [comment]
This issue only occurred Tizen emulator 4.0. Galaxy watch and Tizen emulator 5.0 has no this issue.
I think this issue is emulator issue. not our API issue. 
And CircularUI reference device is real Galaxy watch device not emulator. so do not raise emulator only issue.
